index.js

```diff
@@ -2,16 +2,17 @@ var crypto = require('crypto');
 
 function EncryptedField(Sequelize, key, opt) {
     if (!(this instanceof EncryptedField)) {
-        return new EncryptedField(Sequelize, key);
+        return new EncryptedField(Sequelize, key, opt);
     }
 
     var self = this;
     self.key = new Buffer(key, 'hex');
     self.Sequelize = Sequelize;
-
+    
     opt = opt || {};
     self._algorithm = opt.algorithm || 'aes-256-cbc';
     self._iv_length = opt.iv_length || 16;
+    self._db_encoding = opt.db_encoding || undefined;
```

Comment:
The default is `binary`. Probably makes sense to specify it here; if the default `crypto` package output encoding changes it'd break things in a non-obvious runtime way. Default has to be binary moving forward for compatibility reasons, right?